frontend/src/pages/AccountReceive.vue

```diff
@@ -81,7 +81,7 @@
 </template>
 
 <script lang="ts">
-import { computed, defineComponent, onMounted, ref, watch } from '@vue/composition-api';
+import { computed, defineComponent, onMounted, ref, watch } from 'vue';
```

Comment:
_mostly arbitrary place_

Just manually tested and found this issue:
* on polygon, with advanced mode enabled, on the receive page, I specified a from-block to begin scanning but it still scanned blocks before the specified number

Reply:
This is tracked in https://github.com/ScopeLift/umbra-protocol/issues/275, so going to mark this resolved. Nice job catching it in the QA testing, appreciate the thoroughness 👌